source/vibe/inet/url.d

```diff
@@ -269,6 +281,9 @@ unittest {
 	assert(url.schema == "https", url.schema);
 	assert(url.host == "www.example.net", url.host);
 	assert(url.path == Path("/index.html"), url.path.toString());
+	import std.stdio;
+	if (url.toString != urlstr)
+		writeln(url.toString);
```

Comment:
Oops. Yes, this is debug leftover. We don't needed an adiitional test, because this case tested at the next line: `assert(url.toString == urlstr);`